components/Layout.tsx

```diff
@@ -58,7 +58,7 @@ const useStyles = makeStyles((theme: Theme) =>
       flex: '1 0',
     },
     contentTitle: {
-      margin: theme.spacing(4, 0, 2, 0),
+      margin: theme.spacing(3, 0, 2, 0),
```

Comment:
The top margin looked excessive. Reduced a bit.

Reply:
It is subjective but I felt that was a good amount of top margin, it gave a nice vertical rhythm.

Reply:
Yeah, this is also unrelated to #159 so let's keep it out of this PR.

```suggestion
      margin: theme.spacing(4, 0, 2, 0),
```